CTW/Fourchette/map.json

```diff
@@ -0,0 +1,158 @@
+				{"material": "golden apple", "slot": 7, "amount": 1},
+				{"material": "cooked beef", "slot": 8, "amount": 16},
+				{"material": "arrow", "slot": 9, "amount": 1},
+
+				{"material": "leather helmet", "slot": "helmet", "unbreakable": true},
+				{"material": "chainmail chestplate", "slot": "chestplate", "unbreakable": true},
+				{"material": "leather leggings", "slot": "leggings", "unbreakable": true},
+				{"material": "iron boots", "slot": "boots", "unbreakable": true}
+			]
+		}
+	],
+	"itemremove": [
+		"iron sword", "bow", "iron pickaxe", "iron axe", "iron shovel", "oak planks", "golden apple", 
+		"cooked beef", "arrow", "leather helmet", "chainmail chestplate", "leather leggings", "iron boots"
+	],
+	"killstreaks": [
+		{
+			"count": 1,
+			"repeat": true,
+			"actions": {
+				"items": [
+					{"material": "golden apple", "amount": 1}
+				]
+			}
+		}
+	],
+	"filters": [
+		{
+			"type": "build", "evaluate": "deny", "teams": ["red"],
+			"regions": ["red-spawn-build", "blue-spawn-build", "orange-wool", "lime-wool", "magenta-wool"],
+			"message": "&cYou are not allowed to modify terrain here."
+		},
+		{
+			"type": "build", "evaluate": "deny", "teams": ["blue"],
+			"regions": ["red-spawn-build", "blue-spawn-build", "green-wool", "light-blue-wool", "yellow-wool"],
+			"message": "&cYou are not allowed to modify terrain here."
+		},
+
+		{
+			"type": "enter", "evaluate": "deny", "teams": ["blue"],
+			"regions": ["red-spawn-protection", "green-wool", "light-blue-wool", "yellow-wool"],
+			"message": "&cYou may not enter this region."
+		},
+		{
+			"type": "enter", "evaluate": "deny", "teams": ["red"],
+			"regions": ["blue-spawn-protection", "orange-wool", "lime-wool", "magenta-wool"],
+			"message": "&cYou may not enter this region."
+		},
+
+		{"type": "build", "evaluate": "deny", "teams": ["red", "blue"], "regions": ["build-height"], "message": "&cYou have reached the max build height."}
+	],
+	"regions": [
+		{"id": "red-spawn-protection", "type": "cuboid", "min": "126, 0, 168", "max": "152, oo, 199"},
+		{"id": "blue-spawn-protection", "type": "cuboid", "min": "-8, 0, 198", "max": "-34, oo, 167"},
+		
+		{"id": "red-spawn-build", "type": "cuboid", "min": "126, 0, 168", "max": "152, 10, 199"},
+		{"id": "blue-spawn-build", "type": "cuboid", "min": "-8, 0, 198", "max": "-34, 10, 167"},
+
+		{"id": "green-wool", "type": "cuboid", "min": "-48, 0, 263", "max": "-82, oo, 288"},
+		{"id": "light-blue-wool", "type": "cuboid", "min": "-100, 0, 201", "max": "-127, oo, 164"},
+		{"id": "yellow-wool", "type": "cuboid", "min": "-78, 0, 106", "max": "-43, oo, 75"},
+
+		{"id": "orange-wool", "type": "cuboid", "min": "199, 0, 263", "max": "165, oo, 289"},
+		{"id": "lime-wool", "type": "cuboid", "min": "167, 0, 106", "max": "201, oo, 74"},
+		{"id": "magenta-wool", "type": "cuboid", "min": "219, 0, 167", "max": "253, oo, 202"},
+
+		{"id": "build-height", "type": "cuboid", "min": "-oo, 40, -oo", "max": "oo, oo, oo"}
```

Comment:
You already have the building height defined below this so I'm not sure if you're going to need the filter and region for this.

Reply:
Ha, I see what I did. I used the JSON file from Bloody Trident for this map, seems like the duplicate build height was a mistake on [my part](https://github.com/Warzone/Maps/pull/371/files). I'll fix it right up, as well as [Bloody Trident](https://github.com/Warzone/Maps/pull/381).

The map should be pruned - grabbed it off of ResourcePile. Not sure why it's showing the units in MB.